core/src/main/c/share/dedup.cpp

```diff
@@ -667,6 +667,34 @@ Java_io_questdb_std_Vect_dedupMergeVarColumnLen(JNIEnv *env, jclass cl,
     return dst_var_offset;
 }
 
+
+JNIEXPORT jlong JNICALL
+Java_io_questdb_std_Vect_dedupMergeVarcharColumnLen(JNIEnv *env, jclass cl,
+                                                   jlong merge_index_addr,
+                                                   jlong merge_index_size,
```

Comment:
this parameter should be consistently called "merge_index_row_count", its value is literally derived from

```
timestampMergeIndexSize / TIMESTAMP_MERGE_ENTRY_BYTES
```

we cannot carry on calling this size after this division

Reply:
Ok, done